charts/gardener/gardener-resource-manager/templates/deployment.yaml

```
@@ -0,0 +1,86 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gardener-resource-manager
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: gardener-resource-manager
+    helm.sh/chart: gardener-resource-manager
+    app.kubernetes.io/instance: {{ .Release.Name }}
```

Now, that we move this into g/g, shall we harmonize the labels with the other components? E.g.:
```suggestion
    app: gardener
    role: resource-manager
```

Don't know where the chart is actually used productively though, and if we even still need it.